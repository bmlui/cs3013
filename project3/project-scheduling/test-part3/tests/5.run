./scheduler 0 RR tests/5.in 4
