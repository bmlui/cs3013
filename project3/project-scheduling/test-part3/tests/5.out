Execution trace with RR:
t=0: [Job 0] arrived at [0], ran for: [1]
t=3: [Job 1] arrived at [3], ran for: [4]
t=7: [Job 1] arrived at [3], ran for: [1]
End of execution with RR.
